compose.env

```diff
@@ -17,7 +17,7 @@ UNSTRUCTURED_IO_URL=http://unstructured-io-parsers:9500/
 
 ## COGNITA_FRONTEND VARS
 COGNITA_FRONTEND_PORT=5001
-VITE_QA_FOUNDRY_URL=http://localhost:8000
+VITE_QA_FOUNDRY_URL=http://cognita-backend:8000
```

Comment:
This is probably not correct, localhost was correct, because the host browser cannot resolve `cognita-backend`

Reply:
Will do this change,